fdbcli/TenantCommands.actor.cpp

```diff
@@ -489,6 +478,13 @@ ACTOR Future<bool> tenantListCommand(Reference<IDatabase> db, std::vector<String
 				std::string errorMsgStr = wait(getSpecialKeysFailureErrorMessage(tr));
 				fmt::print(stderr, "ERROR: {}\n", errorMsgStr.c_str());
 				return false;
+			} else if (e.code() == error_code_tenants_disabled /* find more and add them */) {
```

Comment:
Rather than filter the errors here, you could follow the pattern of `tenantGetCommand` and re-catch any errors thrown by `onError`.